translations.txt

```diff
@@ -390,3 +390,232 @@ This will reboot the system=
 This will shutdown the system=
 This will start the array=
 This will stop the array=
+
+; UNRAID.NET / My Servers
+Get Started=
+Sign In=
+Sign Up=
+Sign Out=
+Error=
+Fix Error=
+Close Launchpad and continue to webGUI=
+Learn more=
+Pop-up=
+Back to %s=
+Close %s=
+Close=
+Contact Support=
+LAN IP %s=
+Purchase Key=
+Upgrade Key=
+Recover Key=
+Replace Key=
+Extend Trial=
+%s Avatar=
+Confirm closure then continue to webGUI=
+Close dropdown=
+Open dropdown=
+Please confirm closure You have an open popup=Please confirm closure. You have an open pop-up.
+Trial has expired, see options below=
+Opens %s in new tab=
+My Servers Dashboard=
+Unraid Forums=
+Settings > Management Access • Unraidnet=Settings > Management Access • Unraid.net
+Trial key expires at %s=
+Trial expires in %s=
+Trial key expired at %s=
+Trial expired %s=
+Server up since %s=
+Uptime %s=
+My Servers=
+beta=
+Unraid API Error=
+Failed to connect to Unraid API=
+My Servers Error=
+Please wait a moment and reload the page=
+Close Details=
+Loading My Servers data=
+Displaying last known server data=
+Current server=
+Local access=
+Server Offline=
+Remote access=
+Access unavailable=
+Opens new HTTPS window to %s=
+Sign In to resolve=
+Sign In to Purchase Key=
+Purchase Key or Extend Key=
+No Keyfile=
+Thanks for supporting Unraid=Thanks for supporting Unraid!
+Lets unleash your hardware=Let's unleash your hardware!
+You are all set 👍=
+Sign in or sign up to get started=
+Trial=
+Thank you for choosing Unraid OS=
+Your Trial key includes all the functionality and device support of a Pro key=
+After your Trial has reached expiration, your server still functions normally until the next time you Stop the array or reboot your server=After your *Trial* has reached expiration, your server **still functions normally** until the next time you Stop the array or reboot your server
+At that point you may either purchase a license key or request a Trial extension=At that point you may either purchase a license key or request a *Trial* extension
+You have %s remaining on your Trial key=You have **%s** remaining on your Trial key
+Trial Expired=
+Your Trial has expired=
+To continue using Unraid OS you may purchase a license key=
+You have used all your Trial extensions=You have used all your Trial extensions.
+Alternately, you may request a Trial extension=
+Basic=
+Plus=
+Pro=
+GUID Error=
+Registration key  GUID mismatch=Registration key / GUID mismatch
```

Comment:
Double spaces don't work.  Need to be compressed to one on the left.  Right side can have doubles

Reply:
Tom mentioned that only **helptext.txt** is included by default for English. So I took that as needing to include the default text for my translations inside our PLG like so: `_('Registration key / GUID mismatch')`

Will the translation filtering automatically remove the space and the slash? or just the slash as I've already assumed?

Meaning can I get away with _just_ removing the extra space in this key?

Reply:
I would think you need to remove both.   While I never tested it with space slash space the spec does say no double spaces.   I guess the order in which the parser does things dictates this.

TBH when I went through this chore in the plugins I tested each and every line with a sample translation.